Classes/Domain/Finisher/SaveEventFinisher.php

```diff
@@ -190,13 +182,12 @@ protected function processSysFileReferenceTable(
         $sysFileUid = $databaseData['uid_local'] ?? 0;
 
         // Delete previously stored file references
-        $this->databaseConnection->delete(
+        $this->connectionPool->getConnectionForTable('sys_file_reference')->delete(
```

Comment:
It's really a suggestion to move the connectionPool->getConnectionForTable to a seperate getter function.